_includes/demos/balance-csharp.cs

```diff
@@ -66,7 +58,6 @@ public async Task StartListen()
                 do
                 {
                     result = await this.ws.ReceiveAsync(new ArraySegment&lt;byte&gt;(buffer.Array), CancellationToken.None);
```

Comment:
Shouldn't this be `ArraySegment<byte>` ? Looks like it's using generics or is it something else that I don't know? 🤔